### PR TITLE
update nim.fish sample prompt

### PR DESCRIPTION
**Pull request by nim65s**
Hi,

## Description

This PR updates the sample fish prompt I contributed 6 years ago.

- Use an helper function to wrap informations
- Add VIRTUAL_ENV infos, if any
- Add __fish_git_prompt, wrapped for the theme
- Add comments
- Remove ASCII failback symbols for tty
  (no more useful for me, but if someone really needs it, just ask)

### CODE REVIEW

File: share/tools/web_config/sample_prompts/nim.fish [hunk 1]
```diff
@@ -31,53 +66,41 @@ function fish_prompt
...
-    end
+    # Date
+    _nim_prompt_wrapper $retc '' (date +%X)
+
+    # Virtual Environment
+    set -q VIRTUAL_ENV
+    and _nim_prompt_wrapper $retc V (basename "$VIRTUAL_ENV")
+
+    # git
+    test -d .git
```
Comment (faho) [inline share/tools/web_config/sample_prompts/nim.fish]: `__fish_git_prompt` already figures out if there's a git repo, so this is unnecessary.

Comment (nim65s) [reply share/tools/web_config/sample_prompts/nim.fish]: I need it to know if I have to put the extra `─[]` or not.
In other words, if I remove it, the minimalist prompt would be:
```
┬─[nim@Hattori:~]─[11:39:00]─[]
╰─>$ echo here
```

Comment (faho) [reply share/tools/web_config/sample_prompts/nim.fish]: In that case, you'll want to `test -n` on the output of `__fish_git_prompt`.

Even if there is a `.git` directory, that's not a guarantee that there's an actual git repo - just run `mkdir .git` somewhere.

Comment (nim65s) [reply share/tools/web_config/sample_prompts/nim.fish]: right, thanks :)

Comment (nim65s) [reply share/tools/web_config/sample_prompts/nim.fish]: arg, it doesn't work, and gives `─[G:]`, did I missed something ?

Comment (nim65s) [reply share/tools/web_config/sample_prompts/nim.fish]: fixed :)

**Comment by faho** [comment]
Merged, thank you very much!

**Pull request merged**